src/wp-includes/load.php

```diff
@@ -162,10 +162,11 @@ function wp_check_php_mysql_versions() {
 		exit( 1 );
 	}
 
+	// This runs before default constants are defined, so we can't assume WP_CONTENT_DIR is set yet.
+	$wp_content_dir = defined( 'WP_CONTENT_DIR' ) ? WP_CONTENT_DIR : 'wp-content';
```

Comment:
```suggestion
	$wp_content_dir = defined( 'WP_CONTENT_DIR' ) ? WP_CONTENT_DIR : ABSPATH . 'wp-content';
```

I'd like prepend it to the fallback value instead. Any thoughts?